decidim-admin/spec/features/participatory_process_admin_manages_participatory_process_admins_spec.rb

```diff
@@ -25,6 +25,7 @@
       switch_to_host(organization.host)
       login_as user, scope: :user
       visit decidim_admin.participatory_process_path(participatory_process)
+      click_link "Process admins"
```

Comment:
Using a method called `withing_process_menu` then I cannot use it in a `before`, not sure if I like the option. Also, this method would need to trigger a click and then check for things in the new page, I'm not sure I like it. What about just doing something like this?

```ruby
within "#tabs-panel" do
  click_link "Process admins"
end
```